.travis.yml

```diff
@@ -34,6 +40,9 @@ jobs:
               # Build packages too, this should be done by beaver:
               # https://github.com/sociomantic-tsunami/beaver/issues/34
               - beaver dlang make pkg
+              - if test -n "$TRAVIS_TAG"; then
```

Comment:
This copies only the tagged versions? Could the cache be used as the artefacts storage for the users to download packages? If so, can we lift this, so we can download dirty packages? If not, disregard my note.

Reply:
> This copies only the tagged versions?

This copies only when building a tag, this is because it's useless to upload the cache if the next stage won't run anyway.

> Could the cache be used as the artefacts storage for the users to download packages?

Not that I know of, this is uploaded somewhere in S3 by Travis and I don't know how long does it last. Is something internal to Travis.

Reply:
All right then, if we can't download debian packages manually, it's not worth to upload them for non-tagged buidls.

Reply:
This actually adds about 7s to the build time on every build (time it takes to download and install travis cache utilities), but I guess it is worth it.